### PR TITLE
Release version 2.12.1

### PR DESCRIPTION
**Pull request by rcurtin**
This automatically-generated pull request adds the commits necessary to make the 2.12.1 release.

Once the PR is merged, mlpack-bot will tag the release as HEAD~1 (so that it doesn't include the new HISTORY block) and publish it.

Or, well, hopefully that will happen someday.

### CODE REVIEW

**Comment by coatless** [comment]
@rcurtin / @zoq could this get merged in? I'd like to update the CRAN package for RcppEnsmallen.

File: HISTORY.md [hunk 1]
```diff
@@ -2,13 +2,19 @@
...
     ([#181](https://github.com/mlpack/ensmallen/pull/181)).
 
  * Handle SpSubview_col and SpSubview_row in Armadillo 9.870
     ([#194](https://github.com/mlpack/ensmallen/pull/194)).
 
+ * Minor documentation fixes
+    ([#197](https://github.com/mlpack/ensmallen/pull/197)).
+
+
+
```
Comment (zoq) [inline HISTORY.md]:
```suggestion
```

Comment (zoq) [reply HISTORY.md]: Remove the extra lines.

**Comment by rcurtin** [comment]
Thanks for the rebase; do you mind waiting on this one?  I want to try to make some changes to mlpack-bot.  I'll go ahead and merge it when I think they're ready.

**Comment by rcurtin** [comment]
Okay, let's see what happens...

**Pull request merged**